ports/libjson/README.md

```diff
@@ -0,0 +1,83 @@
+### Tested for QNX 7.1 and 8.0 SDPs
+Cross-compiled on Ubuntu 24.04 for:
+- QNX 8.0 aarch64le on Raspberry Pi 4
+- QNX 7.1 x86_64 on VirtualBox 6.1
+
+__*IMPORTANT NOTE*__: The author of libjson recommends *against* building it as a shared object and linking against it. As of 12/10/2024, the author recommends including libjson in your project directly (per the README).
+As such, these files are intended for testing purposes only. libjson works in QNX with no modifications, and can be found here to include as a submodule in your project: https://github.com/vincenthz/libjson
+
+Instructions for compiling and running tests are listed below.
+
+# Compiling libjson tests for QNX SDP 7.1/8.0
+If you wish to compile in our Docker container, make sure Docker is installed: Requires Docker: https://docs.docker.com/engine/install/
+
+
+1. Make a new workspace or navigate to an existing one
+```bash
+mkdir libjson_wksp && cd libjson_wksp
+```
+
+2. Clone the `libjson` and `build-files` repos
+```bash
+#Pick one:
+#Via HTTPS
+git clone https://github.com/qnx-ports/build-files.git
+git clone https://github.com/vincenthz/libjson.git
```

Comment:
Might want to consider cloning a specific commit so that this doesn't break down the line and so that the version number is alway 1.0.0

Reply:
Added version detection and specified tested commit in readme with f633c68e84cd9de2085ee3e40d3c91eb74726e83 :)